src/components/plots/barPlot.ts

```diff
@@ -142,6 +159,31 @@ export module Plots {
       }
     }
 
+    public addDataset(dataset: Dataset) {
+      if (this._isSetup) {
+        var labelArea = this._renderArea.append("g").classed(Bar._LABEL_AREA_CLASS, true);
```

Comment:
in `addDataset()` on the parent `Plot` level there's a statement that uses `if (this._isSetup)`.  I could put the setup in there, or leave it here.